src/main/java/com/amihaiemil/camel/YamlSerializer.java

```diff
@@ -0,0 +1,39 @@
+package com.amihaiemil.camel;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * A Yaml serializer.
+ * @author sherif
```

Comment:
@SherifWaly Here it should be your name and email address between parentheses (see how it's in other classes)